airbyte-integrations/connectors/source-shopify/source_shopify/schemas/orders.json

```diff
@@ -263,9 +263,6 @@
     "fulfillment_status": {
       "type": ["null", "string"]
     },
-    "gateway": {
```

Comment:
As you wish! Is this something you usually do when updating Airbyte catalogs to match source schema changes? IMHO it feels strange to keep accumulating "dead" columns, and it still introduce a "breaking change" for end-user depending on what they do with these fields (which will now be empty)

Reply:
I think we should remove these if the API no longer returns them

Reply:
Just to avoid breaking change @sherifnada, if this is acceptable for all the rest of the users - I'm also fine.

Reply:
the spirit of making field removal backwards breaking is that it forces us to inform users of this shift and allows them to prepare for it. Not removing it just "hides" that these fields were removed and users might get incorrect analyses downstream since they weren't made aware of these changes. So I think if we know the fields will no longer get populated we should make the breaking change

Reply:
As stated above, I do agree. These fields will be changed once we update (going from having value to being empty), so these are litteraly breaking changes anyway.

Reply:
Having said that I think a breaking change will likely require informing users and giving them a timetable to migrate so this change will likely take a couple of weeks to give users time to adjust

Reply:
which means we cannot include this changes into GA certification @lazebnyi @oustynova

Reply:
I'm quite new to Airbyte, but I was under the impression breaking changes communication were handled at the product level (https://docs.airbyte.com/cloud/managing-airbyte-cloud/manage-schema-changes/) with webhook and UI notification to let users review changes and resume syncs. That's not how it works? (Part of this PR was to test that breaking change behavior)

If this is not the case, and that handling breaking changes requires a long process, we might have an issue with Shopify versioning policy because they include breaking changes every 3 months: https://shopify.dev/docs/api/usage/versioning
In addition, the currently used api version (2022-10) will be discontinued in October, so Airbyte API calls will automatically use the oldest stable version (2023-04).

Anyway, considering this PR, as a new quarter version (2023-07, currently release candidate) is supposed to be released tomorrow, we might wait and upgrade directly to this one, WDYT?

Reply:
upgrading to 2023-07 instead makes a lot of sense to me

Reply:
Let do it @loris